test/integration/proxy_test.js

```diff
@@ -715,7 +715,7 @@ test('reroutes retry to remote', function t(assert) {
             retrySchedule: [1]
         }, function onRequest(err, res) {
             assert.ifError(err, 'no error occurs');
-            assert.equal(res.body, 'rerouted', 'response from rerouted request is correct');
+            assert.equal(res.body.toString(), 'rerouted', 'response from rerouted request is correct');
```

Comment:
The API changed to not convert Buffers to strings automatically.

The `res.body` is a `Buffer`, the performance optimization is not to convert it to a string.

I can do `toString('utf8')` to make it more clear, Or I can do: 

```
assert.equal(res.body.toString('hex'), new Buffer('rerouted').toString('hex'))
```